content/docs/use-cases/model-registry.md

```diff
@@ -0,0 +1,52 @@
+# Machine Learning Model Registries
+
+A **model registry** is a central catalog for data artifacts produced in
+multiple data science projects. Models can be discovered, audited, shared,
+tested, and deployed from here. DVC works on top of Git to enable these
+capabilities on your existing ecosystem. And other tools from
+[Iterative](https://iterative.ai/) can provide further advanced features and
+better usability as well.
+
+![](/img/ml_model_registry_placement.jpg) _A common placement for model
+registries is between development and applications. See also [Data Registries]._
+
+[data registries]: /doc/use-cases/data-registries
+
+DVC model registries give your team key collaborative capabilities:
+
+- Store, organize, and track the [versions] of different models effectively.
+- Full data provenance and lineage trails guarantee that your modeling is
+  reproducible.
+- Share and control access to early-stage [experiments] and production-ready
+  models alike.
+- Save performance [metrics and plots] to quickly evaluate and compare models.
+- A platform to export, integrate, deploy, or roll-back published models
+
+[experiments]: /doc/user-guide/experiment-management
+[metrics and plots]: /doc/start/metrics-parameters-plots
+
+Many of these benefits are built into DVC: Your [modeling process] and
+performance metadata become **codified** in Git-based <abbr>DVC projects</abbr>.
+This makes it possible to manage them with standard GitOps workflows. Large
+files can be pushed to [remote storage], an efficient and scalable access point
+for your models.
+
+To achieve the remaining goals, the final outputs of these model development
```

Comment:
At this moment we think that Studio should be solving this - combining models from multiple repos.

Saving performance metrics, evaluate and compare - this can be done in a single repo with DVC or in Studio

`A platform to export, integrate, deploy, or roll-back published models` - this is done by `gto` (which is published already), by `mlem` (deploy, and it's coming), `roll-back` (also by GTO). Btw this item is a bit too complicated. In our case this system provide interface to access a model of a specific version (on CI, or from a service). ... other tools do more things

Reply:
> we think that Studio should be solving this - combining models from multiple repos

Yep but that release is still a bit away I think?

We could focus on a hypothetical manual registry done purely with DVC for now. We'll definitely want to revisit this use case soon anyway (it's in the Q2 plan already).

---

Working on the GTO mentions... ⌛

Reply:
> this item is a bit too complicated. In our case this system provide interface to access a model of a specific version (on CI, or from a service)

Simplified for now.